### PR TITLE
Basic project setup

### PR DESCRIPTION
**Pull request by pooja-ces**

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const Footer: React.FC = () => {
+    return (
+        <>
+            <div className='mt-10 px-20 fixed bottom-0 w-full border-t py-3 bg-white'>
```
Comment (aashikvilla) [inline src/components/Footer.tsx]: If you have decided to add component as part of this PR, lets also cover the tests.
Ensure you have tests for any piece of code.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                <div className='flex justify-between items-center'>
+                    <div>
+                        <ul className='flex items-center gap-3'>
+                            <li className=' decoration-0 no-underline'>About</li>
```
Comment (aashikvilla) [inline src/components/Footer.tsx]: Remove extra space  in class name.
The styles look repetitive, probably move it to parent ?

File: src/components/Footer.tsx (hunk 1 above)
Comment (pooja-ces) [reply src/components/Footer.tsx]: I have added

File: src/components/Footer.tsx (hunk 2 above)
Comment (pooja-ces) [reply src/components/Footer.tsx]: fixed

File: src/__tests__/Footer.test.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import '@testing-library/jest-dom';
+import Footer from '../components/Footer';
+
+describe('Footer Component', () => {
+    it('renders the footer with links', () => {
+        render(<Footer />);
+
+        const aboutLink = screen.getByText(/About/i);
+        const termsLink = screen.getByText(/Terms/i);
+        const privacyPolicyLink = screen.getByText(/Privacy Policy/i);
```
Comment (aashikvilla) [inline src/__tests__/Footer.test.tsx]: Better to check for exact match than case insensitive

**Review by aashikvilla — APPROVED** [review]

File: src/__tests__/Footer.test.tsx (hunk 3 above)
Comment (pooja-ces) [reply src/__tests__/Footer.test.tsx]: Fixed

**Pull request merged**